sig-auth/annual-report-2021.md

```diff
@@ -0,0 +1,114 @@
+# Kubernetes SIG Auth 2021 Annual Report
+
+## Operational
+
+**How are you doing with operational tasks in sig-governance.md?**
+
+**Is your README accurate? have a CONTRIBUTING.md file?**
+
+ - Yes, our README is accurate. As of this report, we do not have a CONTRIBUTING.md file. We will be creating one shortly.
```

Comment:
Do we have a tracking k/k issue for this?

Reply:
Issue to track: https://github.com/kubernetes/community/issues/5760